check-formatting.sh

```diff
@@ -9,16 +9,17 @@ RESET=$(tput sgr0)
 
 npm install &>/dev/null
 
+PROTO_OPTIONS="--proto_compiler //external:proto_compiler --proto_toolchain_for_java //external:proto_java_toolchain"
```

Comment:
Do we still need these if we add them to `bazelrc`?

(throughout this PR)

Reply:
@oferb I think we don't